### ISSUE TITLE
Never wipe internal artifacts

### ISSUE THREAD

**Issue by grtjn** [marklogic/roxy#818]
You might want to reuse something that comes with MarkLogic out of the box, like Schemas db, but that should not get removed once you wipe your project.

This would apply to everything that comes with MarkLogic. E.g.:

- groups: Default
- app-servers: Admin, App-Services, HealthCheck, Manage
- databases: App-Services, Documents, Extensions, Fab, Last-Login, Meters, Modules, Schemas, Security, Triggers
- forests: App-Services, Documents, Extensions, Fab, Last-Login, Meters, Modules, Schemas, Security, Triggers
- users: admin, healthcheck, infostudio-admin, nobody
- roles
- privs
- collections
- certs
- amps
- mimes

Basically anything Roxy can bootstrap. Databases, forests, users, and roles are probably most important.

**Comment by grtjn** [comment]
I also created this ticket because I was about to add a lot of checks like: don't insert test-modules-db if test-modules-db = "Modules" (or any other built-in actually), and this prevents the need for a lot of those checks.

**Comment by grtjn** [comment]
Bootstrapping should be okay. Things won't get created if they already exist, which is good. And one may want to apply settings on Admin (to enable https), Documents (add indexes for some ad hoc test), or whatever.
